components/fxa-client/src/attached_client.rs

```diff
@@ -0,0 +1,32 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+pub use crate::http_client::GetAttachedClientResponse as AttachedClient;
+use crate::{error::*, util, CachedResponse, FirefoxAccount};
+
+// An attached client response is considered fresh for `ATTACHED_CLIENTS_FRESHNESS_THRESHOLD` ms.
+const ATTACHED_CLIENTS_FRESHNESS_THRESHOLD: u64 = 60_000; // 1 minute
+
+impl FirefoxAccount {
```

Comment:
It's a good idea putting this impl in a different module. One nit though, could we put this in the `oauth::attached_client` namespace since it is OAuth related?

Reply:
@eoger before I do anything unnecessary, did you mean to move this logic to oauth.rs as static functions or within the pre-existing FirefoxAccount implementation? I was initially thinking to move attached_client.rs to a new oauth folder but I forgot about the existing oauth.rs file and module.

Reply:
> @eoger I was initially thinking to move attached_client.rs to a new oauth folder but I forgot about the existing oauth.rs file and module.

Yup that's what I was thinking too!
I believe creating an `oauth/attached_client.rs` file "should" work since there's no need for `mod.rs` files anymore in rust 2018 edition.

Reply:
Oh, nice--okay, thanks!!